registro

SergCloud0 opened the pull request:
acá eh colocado nombre del perfil como registro

SergCloud0 committed:
registro

acá eh colocado nombre del perfil como registro

appsmambo reviewed (commented):
Hay problemas con los caracteres especiales, tildes y eñes.

SergCloud0 committed:
Update grupal.psc

Eh corregido el nombre de registro
y borre nombres antes de la opción 7
y los caracteres especiales.

appsmambo reviewed (approved):
ok muchas gracias.
